google/cloud/storage/internal/read_object_range_request_test.cc

```diff
@@ -17,6 +17,16 @@
 
 using namespace storage::internal;
 
+namespace {
+HttpResponse CreateRangeRequestResponse(char const* content_range_header) {
+  HttpResponse response;
+  response.headers.emplace(std::string("content-range"),
```

Comment:
Similar to above comment, this should probably be "Content-Range".

To that tune, do you think it's worth defining header names as constants somewhere, and referencing them from their variable names?  This would help prevent typos (both from misspellings and case sensitivity).

Reply:
We should have some set of "well known headers", yes.  OTOH, the tests are supposed to catch that anyway.

I am not going to change the spelling right now, because it would break.